docs-source/spring/content/development/cli/_index.md

```diff
@@ -30,26 +30,30 @@ Short descriptions of the available commands are as follows.
 
 ## AVAILABLE COMMANDS
 
-Built-In Commands
+Admin Server Commands
+   - `change-password`: change password for Oracle Spring
+connect: connect to the Oracle Spring admin console
+
+Application/Namespace Commands
+   - `create`: create an application/namespace
+   - `delete`: delete a service or entire application/namespace
 
-- `help`: Display help about available commands
-- `stacktrace`: Display the full stacktrace of the last error.
-- `clear`: Clear the shell screen.
-- `quit`, `exit`: Exit the shell.
-- `history`: Display or save the history of previously run commands
-- `version`: Show version info
-- `script`: Read and execute commands from a file.
-
-Commands
-
-- `connect`: connect to the Oracle Spring admin console
-- `change-password`: change password for Oracle Spring
-- `create`: create an application/namespace
-- `create-schema`: create a schema/user and bind it to service deployment
-- `config`: view and modify application configuration
-- `deploy`: deploy a service
-- `list`: list/show details of application services
-- `delete`: delete a service or entire application/namespace
+Built-In Commands
+   - `help`: Display help about available commands
+   - `stacktrace`: Display the full stacktrace of the last error.
+   - `clear`: Clear the shell screen.
+   - `quit`, `exit`: Exit the shell.
+   - `history`: Display or save the history of previously run commands
+   - `version`: Show version info
+   - `script`: Read and execute commands from a file.
+
+Informational Commands
+   - `list`: list/show details of application services
```

Comment:
the descriptions should have consistent formatting. some start with a cap, some don't. some end with a period, some don't.